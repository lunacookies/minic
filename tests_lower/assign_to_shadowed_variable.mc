func main {
	a := 5
	a := &a
	b := 10
	set a = &b
}
