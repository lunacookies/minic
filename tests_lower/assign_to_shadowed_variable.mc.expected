func main
	var a i64
	var b i64
	{
		set a = 5
		<missing>
		set b = 10
		<missing>
	}
tests_lower:21..28: error: cannot shadow existing variable
tests_lower:47..49: error: expected “i64” but found “*i64”
tests_lower:13..19: warning: unused variable
